src/Common/src/Interop/OSX/System.Native/Interop.ProtocolStatistics.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+	internal static partial class Sys
+	{
+		[StructLayoutAttribute(LayoutKind.Sequential)]
+		public unsafe struct TcpGlobalStatistics
+		{
+		    public readonly ulong ConnectionsAccepted;
+		    public readonly ulong ConnectionsInitiated;
+		    public readonly ulong ErrorsReceived;
+		    public readonly ulong FailedConnectionAttempts;
+		    public readonly ulong SegmentsReceived;
+		    public readonly ulong SegmentsResent;
+		    public readonly ulong SegmentsSent;
+		}
+
+		[DllImport(Libraries.SystemNative)]
```

Comment:
Should we have a System.Net.Native, or are we ok having all of this in System.Native?